### DISCUSSION TITLE
Hiding Measurement widget in 3.0

### DISCUSSION THREAD

**Discussion by kmurari103** [iTwin/itwinjs-core#3368] [Q&A]
Hi there ,
I found that in 3.0, there is no prop to hide Measurement widget.
If i do hide it this way, then measure tool (at corner left side) also gets hidden. Is there a way i just hide widget but keep the tool visible?

```
defaultUiConfig={{
          hidePropertyGrid: true,
          hideTreeView: true,
          contentManipulationTools: { 
                     // this hides/shows both measurement widget as well as measure tool
                    verticalItems: { measureTools: true/false }
                    },
        }}
```
![image](https://user-images.githubusercontent.com/27068008/158589118-87756989-2d63-4e46-a309-921ef6f82f7d.png)

**Comment by aruniverse** [comment]
Is there a specific reason you don't want the widget? Once a measurement has been created or selected it will show the properties for that measurement.

**Reply by kmurari103** [reply]
Its just that earlier i had only application specific widgets displayed but now Measurements widget is also there.
Although  I did use measure tool to see any measurement related stuffs earlier but not the widget.

**Reply by aruniverse** [reply]
Hi, the measure-tools-react pkg has been updated to hide this widget by default and only show when a measurement is made in v0.7.0
